docs/classic-vs-vt.md

```diff
@@ -51,14 +51,16 @@ As Windows has evolved, the security controls and restrictions on window handles
 
 ### Unicode
 
-The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist.  However, when otherwise undefined, using UTF-8 is widely accepted as the appropriate default. It represents the balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs.
+The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist. However, when otherwise undefined, using UTF-8 is widely accepted as the appropriate default. It represents the balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs. However, an algorithmic equivalent in UTF-16 was chosen as the primary Unicode support mechanism for the Windows platform.
 
-The Windows console platform has supported and will continue to support all existing code pages and encodings, but we recommend UTF-8 for all forward looking development and also accept UTF-16 as an algorithmically-translatable alternative.
-
-UTF-8 support in the console can be found via the _A_ variant of all Console APIs against console handles after setting the codepage to `65001` or `CP_UTF8` with the [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods as appropriate. Setting the code pages in advance is only necessary if the machine has not chosen "Use Unicode UTF-8 for worldwide language support" in the settings for Non-Unicode applications in the Region section of the Control Panel.
+The Windows console platform has supported and will continue to support all existing code pages and encodings. It is recommended to focus on UTF-16 for maximum compatibility across Windows versions and performing algorithmic translation with UTF-8 if necessary. Increased support of UTF-8 is in progress for the console system.

```

Comment:
It is recommended _that you use UTF-16_ (rather than "to focus on"), `and perform` (not "performing")

Reply:
done.